app.py

```diff
@@ -89,7 +116,7 @@ def main():
             ratingsRDD = ratingsRDD.map(lambda x: (x[0], x[1], x[2]))
 
             model_version += 1
-            print model_version
+            logger.info("model version"=model_version)
```

Comment:
this needs to be a formatted string, so something like `"model version = {}".format(model_verison)`